docs/http/http-spans.md

```diff
@@ -644,6 +649,38 @@ and SHOULD be provided **at span creation time** (if provided at all):
 
 `http.route` MUST be provided at span creation time if and only if it's already available. If it becomes available after span starts, instrumentation MUST populate it anytime before span ends.
 
+## Capturing HTTP headers in traces
+
+In addition to the attributes defined in this document, HTTP client and server instrumentations MUST also allow
```

Comment:
I don't think we can use MUST here

1. there are dozens of instrumentations out there that would become non-conformant.
2. we never require configurability and we can't until it's stable and adopted by language SDKs

We already define request and response attributes - search for `http.request | response.header.<key>` and we say that instrumentations MAY provide config to capture them. I don't think we can be stronger than that.